SCF-Session6-Question1/src/meta/utility/IntSet.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * This class defines structure for IntSet 
+ * This is Service Class
+ * Where this class is return values as per requested service 
+ * @author ABHISHEK BISHNOI
+ * @version 1.0
+ * @since 22-July-2019
+ */
+package meta.utility;
+
+import java.util.LinkedHashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+public class IntSet implements IntSetInterface{
+	private final Set<Integer> set = new LinkedHashSet<Integer>();
```

Comment:
The problem states that we have to implement IntSet class using an array. We can't use java.util.Set and its in built functions for this problem.